gui/src/2D/valueAndUnit.ts

```diff
@@ -75,7 +75,7 @@ export class ValueAndUnit {
      * @returns the value
      */
     public getValue(host: AdvancedDynamicTexture): number {
-        if (host && !this.ignoreAdaptiveScaling && this.unit !== ValueAndUnit.UNITMODE_PERCENTAGE) {
+        if (host && !this.ignoreAdaptiveScaling && this.unit !== ValueAndUnit.UNITMODE_PERCENTAGE && host.renderAtIdealSize) {

```

Comment:
This seems incorrect as ideadlWith and height can still be used: when renderAtIdealSize = false;
https://doc.babylonjs.com/divingDeeper/gui/gui#adaptive-scaling

renderAtIdealSize is here to force the texture to rescale always at the expected ideal size